daps_utils/__init__.py

```diff
@@ -1,13 +1,17 @@
-from .__initplus__ import __basedir__, load_config, load_current_version
+import os
 
-try:
-    from .breadcrumbs import drop_breadcrumb as talk_to_luigi
-    from .tasks import (MetaflowTask, CurateTask, ForceableTask,
-                        DapsTaskMixin, DapsRootTask)
-    from .flow import DapsFlowMixin
-    config = load_config()
-except ModuleNotFoundError as exc:  # For integration with setup.py
-    print(exc)
-    pass
+from .__initplus__ import __basedir__, load_config, load_current_version
 
 __version__ = load_current_version()
+
+if "IN_SETUP" not in os.environ:
```

Comment:
The following imports fail if we are being imported from `setup.py`.

Relying on environment variable is a bit messy but is more robust than letting `ImportError`s pass.

Reply:
Doesn't this mean that we don't need the below `try` statement any more? (i.e can unindent and remove the `except`)

Reply:
Ok. I wasn't clear what case the try-except was capturing - I get `ImportError` not `ModuleNotFoundError`